proto/api/v2/compliance_scan_configuration_service.proto

```diff
@@ -13,11 +13,21 @@ option java_package = "io.stackrox.proto.api.v2";
 
 // ClusterScanStatus holds status based on cluster in the event that a scan configuration
 // was successfully applied to some clusters but not others.
-// Next available tag: 3
+// Next available tag: 5
 message ClusterScanStatus {
   string cluster_id = 1;
   repeated string errors = 2;
   string cluster_name = 3;
+
+  // Additional scan status gathered from ComplianceSuite
+  message SuiteStatus {
```

Comment:
I was trying to think of another name for this.  My hope was to abstract away the fact that this is from the compliance suite for anyone calling it via API.  But I was unable to come up with something better.  🤷